tools/runners/test262-harness.py

```diff
@@ -830,7 +830,7 @@ def run(self, command_template, tests, print_summary, full_summary, logname, job
             report_error("No tests to run")
         progress = ProgressIndicator(len(cases))
         if logname:
-            self.logf = open(logname, "w")
+            self.logf = open(logname, "w", encoding="utf8", errors="ignore")
```

Comment:
I don't think that this will work on py2. The `open` builtin of py2 has neither `encoding` nor `errors`. However, the `codecs` builtin module is awailable both in py2 and py3, and might be used to open files with a given encoding.

See:
- https://docs.python.org/2.7/library/codecs.html#codecs.open
- https://docs.python.org/3.9/library/codecs.html#codecs.open

This approach (i.e., `codecs.open`) might be useful everywhere where files are opened, making `.decode()` calls unnecessary.

Reply:
OK, does